ManatiCR/vitaminau
# Feature/vit 13 add webform apoyo emprendimiento

# Task [VIT-13](https://manati.atlassian.net/browse/VIT-20?atlOrigin=eyJpIjoiMDg0NTk3ZjJhNTExNDg2ZjhhYjFkMjQxOTJhMjJkM2UiLCJwIjoiaiJ9)
Creation of the "Apoyo al emprendimiento" webform.

# Installation
- git pull in this branch
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user
  - Username: admin
  - Password: admin
- Go to the following web path: 
  - http://vitaminau.docker/form/tecnologias-licenciadas
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-13](https://manati.atlassian.net/secure/RapidBoard.jspa?rapidView=104&projectKey=VIT&modal=detail&selectedIssue=VIT-26)
- Fill all the fields of the form and press the save button
- Once you press it your information should be saved and the form should redirect you to a thank you page
- if you want check your response you can go this path and search your answer


# Acceptance criteria
- Only authenticated users can access the form.
- The following fields are not required
  - Restricciones
  - ¿Algo que agregar?
  - PDF relevante
  - ¿Qué mejoraría?
  - At the moment the space "unit" is not required
- Section named "Beneficiarios" is temporal section, because it need changes
